packages/docs-app/src/examples/core-examples/progressExample.tsx

```diff
@@ -59,7 +59,7 @@ export class ProgressExample extends BaseExample<IProgressExampleState> {
         ];
     }
 
-    private renderLabel = (value: number) => value.toFixed(1);
+    private labelRenderer = (value: number) => value.toFixed(1);
```

Comment:
this is a method, so I believe it should remain `renderLabel`

Reply:
Yep